File: starters/CSharp/Bomberjam/Bonus.cs
using System.Text.Json.Serialization;

namespace MyBot.Bomberjam
{
    public class Bonus
    {
        [JsonPropertyName("x")]
        public int X { get; set; }

        [JsonPropertyName("y")]
        public int Y { get; set; }

        [JsonPropertyName("kind")]
        public string Kind { get; set; }
    }
}

File: starters/PHP/bomberjam/Bonus.php
<?php

namespace bomberjam;

class Bonus
{
    /** @var int */
    public $x;

    /** @var int */
    public $y;

    /** @var string */
    public $kind;

    public function __construct($jsonBonus)
    {
        $this->x = $jsonBonus['x'];
        $this->y = $jsonBonus['y'];
        $this->kind = $jsonBonus['kind'];
    }
}